testing.rst

```diff
@@ -31,6 +31,17 @@ command:
 
     $ ./vendor/bin/simple-phpunit
 
+.. note::
```

Comment:
What if we rework the part above (where we already list the command) to say something like: “after the library downloads, try executing phpstorm by running: (then the terminal block). The first time you run this, it will download PHPUnit itself and make its classes available in your app.

Reply:
Thanks, I reworded as I you suggest.